wsmaster/wsmaster-local/src/test/java/org/eclipse/che/api/local/LocalSnapshotDaoTest.java

```diff
@@ -80,7 +83,7 @@ private SnapshotImpl createSnapshot() {
         return SnapshotImpl.builder()
                            .generateId()
                            .setType("docker")
-                           .setInstanceKey(new DummyInstanceKey())
+                           .setMachineSource(machineSource)
```

Comment:
yes for sure, but there is a dedicated test class for MachineSource serialization , MachineSourceAdapter + MachineSourceAdapterTest class

so this test will reuse MachineSourceAdapter which is tested